app/views/orders/past_orders.html.erb

```diff
@@ -0,0 +1,58 @@
+<%= render "header" %>
+<div id="full-order-page" class="full-order-page">
+  <div class="row orders present center">
+    <h4>Past Orders</h4>
+  </div>
+
+  <% if @past_orders.any? %>
+
+  <div class="orders row">
+    <div class="order-placeholders col l2"> &nbsp; </div>
+      <div class="row my-orders col l8">
+        <div class="order-header">
+          <div class="col l2">Order Number</div>
+          <div class="col l2">Date</div>
+          <div class="col l2">Total</div>
+          <div class="col l2">Status</div>
+          <div class="col l2">&nbsp;</div>
+          <div class="col l2">Order Details</div>
+        </div>
+      </div>
+    <div class="order-placeholders col l2"> &nbsp; </div>
+  </div>
+
+    <% @past_orders.each do |order| %>
+    <div class="orders row">
+    <div class="order-placeholders col l2"> &nbsp; </div>
+      <div class="row my-orders col l8">
+
+        <div class="order-group">
+          <div class="order-body">
+            <div class="col l2"><%= order.user_id %></div>
+            <div class="col l2">
+              <%= Date.strptime((order.created_at).to_s) %>
```

Comment:
I think this could be better implemented using `decorators`. It's better to use decorators to format complex data for user display